src/main/java/com/box/sdk/BoxUser.java

```diff
@@ -337,11 +337,27 @@ public static BoxUser getCurrentUser(BoxAPIConnection api) {
      * @return       the newly created email alias.
      */
     public EmailAlias addEmailAlias(String email) {
+        return this.addEmailAlias(email, false);
+    }
+
+    /**
+     * Adds a new email alias to this user's account and confirms it without user interaction.
+     * This functionality is only available for enterprise admins.
+     * @param email the email address to add as an alias.
+     * @param isConfirmed whether or not the email alias should be automatically confirmed.
+     * @return the newly created email alias.
+     */
+    public EmailAlias addEmailAlias(String email, boolean isConfirmed) {
         URL url = EMAIL_ALIASES_URL_TEMPLATE.build(this.getAPI().getBaseURL(), this.getID());
         BoxJSONRequest request = new BoxJSONRequest(this.getAPI(), url, "POST");
 
         JsonObject requestJSON = new JsonObject()
-            .add("email", email);
+                .add("email", email);
+
+        if (isConfirmed) {
```

Comment:
How is this different from the addEmailAlias method when isConfirmed is false. Also, I can't find any documentation about it in https://developer.box.com/reference#get-all-email-aliases-for-a-user

Reply:
The `is_confirmed` parameter allows an admin to add an email alias for a user without needing the user to manually verify the email address.  I'll follow up about how we want to document it, but it's definitely something that's in use today.

Reply:
Since is_confirmed defaults to *false* in the `addEmailAlias(String)` method, can we just make the second method simple, something like `addConfirmedEmailAlias(String)` which internally defaults the is_confirmed param as *true*.

Reply:
Is there a way to do that without duplicating logic?  I'm looking at the code and it seems like in order to have two simple methods like that I would need to copy the main logic for the method across both.

Reply:
```
public EmailAlias addEmailAlias(String email)
public EmailAlias addConfirmedEmailAlias(String email)
private EmailAlias addEmailAlias(String email, boolean isConfirmed)
```
Both the public methods can call this private method